lib/Sema/ConstraintGraph.cpp

```diff
@@ -27,7 +27,9 @@
 using namespace swift;
 using namespace constraints;
 
+#if !defined(_MSC_VER)
 #pragma mark Graph construction/destruction
+#endif
```

Comment:
Same comments about pragma mark: if we have to put this nonsense around it every time, it might not be pulling its weight. @DougGregor, @rudkx, @xedin?